Makefile

```diff
@@ -172,6 +172,37 @@ ifeq ($(BUILD_ENTERPRISE_READY),true)
 	docker-compose -f docker-compose.yaml up haproxy
 endif
 
+run-multi: run-client
+ifeq ($(BUILD_ENTERPRISE_READY),true)
+	@echo Removing possibly wrong configured default network
+	docker network rm mattermost-server_mm-test
```

Comment:
Does this need to be done every time we run this or only one time? Because it seems like a one-time thing of removing the networks and letting them be created again.

Reply:
The issue is IP-classing with the non-multi compose file. SInce for this setup we need 2 distinct subnets we need to remove any possible mm-test that was created as part of the traditional server run with `make run` which uses a broader scope of IPs.

The network rm command is super lightweight, so on a second run it will basically be a no-op